docs/setup.md

```diff
@@ -153,7 +180,7 @@ These commands perform the following steps:
 * build basic Isabelle images to ensure that
   the installation works. This may take a few minutes.
 
-Alternatively, it is possible to use the official Isabelle2021 release
+Alternatively, it is possible to use the official Isabelle2022 release
```

Comment:
Would it be better to say "the latest official version of Isabelle", or do we want to come back and check this as part of the Isabelle update every time?

Reply:
The problem is that it's not always the latest version. E.g. when Isabelle2023 comes out it'll likely take a few months until we're updated and during that time the sentence is wrong. Of course when we forget to update it like last time, then the sentence is wrong for a longer time ;-)